api/UserActivation.json

```diff
@@ -11,7 +11,7 @@
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
-            "version_added": false
+            "version_added": "120"
```

Comment:
FYI, this is all correct. `UserActivation` (and `Navigator.userActivation`) added by https://bugzilla.mozilla.org/show_bug.cgi?id=1791079.